docs/tutorials/first_steps/08_create_workflow.rst

```diff
@@ -93,7 +127,9 @@ and other commands.
     .. code-block:: console
 
         $ rm data/output/*
-        $ renku run python src/filter_flights.py data/flight-data/2019-01-flights.csv.zip data/output/flights-filtered.csv
+        $ renku run X filterFlights.X data/flight-data/2019-01-flights.csv.zip data/output/flights-filtered.csv
```

Comment:
This isn't quite sufficient, since in the earlier section, we recommend that the scripts be saved in the `src` folder (which I think is a good idea). More of the path needs to be specified here.

Reply:
Fixed